controllers/datafoundation.go

```diff
@@ -316,3 +324,32 @@ func (r *dataFoundationReconciler) reconcileProviderAPIServerNetworkPolicy() err
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileOCSInitialization() error {
+	r.Log.Info("Reconciling OCSInitialization")
+
+	ocsInitList := ocsv1.OCSInitializationList{}
+	if err := r.list(&ocsInitList, &client.ListOptions{
+		Namespace: r.dfNamespace,
+	}); err != nil {
+		return fmt.Errorf("failed to list OCSInitialization resources: %v", err)
+	}
+	if len(ocsInitList.Items) == 0 {
+		r.Log.V(-1).Info("OCSInitialization resource not found")
+		return nil
+	}
+	isChanged := false
+	obj := &ocsInitList.Items[0]
+	if !obj.Spec.EnableCephTools {
+		isChanged = true
+		obj.Spec.EnableCephTools = true
+	}
+	isChanged = r.addOwnerAnnotation(obj)
+	if isChanged {
+		if err := r.update(obj); err != nil {
+			return err
+		}
+	}
```

Comment:
We do not use owner annotation anymore, that feature suggestion was never merged into main. 
We decided to go with non-controller owner refs
```suggesion
	csv := &ocsInitList.Items[0]
	if err := r.Own(csv, false); err != nil {
		return err
	}
	if !csv.Spec.EnableCephTools {
		obj.Spec.EnableCephTools = csv
		if err := r.update(csv); err != nil {
			return err
		}
	}
```